server/cluster/cluster.go

```diff
@@ -1306,6 +1306,7 @@ func (c *RaftCluster) processRegionHeartbeat(ctx *core.MetaProcessContext, regio
 			func(context.Context) {
 				c.changedRegions <- region
 			},
+			ratelimit.WithRetained(true),
```

Comment:
Would you be able to handle this error in the function?

Reply:
What do you mean?